services/121-service/test/payment/do-payment.scope.test.ts

```diff
@@ -71,16 +76,32 @@ describe('Registrations - [Scoped]', () => {
       scope: testScope,
       adminAccessToken: accessToken,
     });
+    // add scoped user as approver ..
+    const userIdScoped = (
+      await getCurrentUser({ accessToken: accessTokenScoped })
+    ).body.user.id;
+    await createApprover({
+      programId: PvProgramId,
+      userId: userIdScoped,
+      order: 1,
+      accessToken,
+    });
+    // .. and remove default admin-user approver
+    await deleteApprover({
+      programId: PvProgramId,
+      approverId: 1,
+      accessToken,
```

Comment:
Now that I read this I think it's better to have a query param for approvers on the reset endpoint, as suggested in my earlier comments. What do you think?